### ISSUE TITLE
Turn with mouse cursor

### ISSUE THREAD

**Issue by GreenLunar** [supertuxkart/stk-code#1832]
Add an ability to set left/right mouse movement for left/right navigation. Speed Dreams has such implementation as a default.

Speed Dreams - A free Open Motorsport Sim and Open Source Racing Game
http://speed-dreams.org/ http://sourceforge.net/projects/speed-dreams/

**Comment by bananasmoothii** [comment]
Yep, as I said I  that issue mentionned above (#4520), controls could be:
- mouse position compared to initial position: forward, backward, left and right
- left click: skid
- middle click: turbo
- right click: throw / use
- double middle click: pause
- scroll down: see behind
- scroll up: come back to the road if we fell down

Of course a sensibility setting would be great.

**Comment by GreenLunar** [comment]
_> Edit: *** this is from 2014 ?!_

Time flies. GET MARRIED, go to the country side,  and have as many kids as possible (:

**Comment by jwaxy** [comment]
Is there any progress in this? It would be a great feature.

**Comment by qwertychouskie** [comment]
> Is there any progress in this? It would be a great feature.

This is a pretty niche request, so I wouldn't hold your breath for the core dev team to work on it.  If someone make a PR with a good implementation of the feature, I would expect it to be merged, but otherwise, it's unlikely to happen.

**Comment by GreenLunar** [comment]
I disagree.

It is mostly practical.

### Introduction

I've used to play racing video games and also participated in
real-world kart races from time to time, ever since I was 8.

I've played racing video games when I was a child until my teenage
years, mostly Need For Speed 2, 3, 4 and 5.

I was qualified to flight a Cessna when I was 17, in less than 6 months
since I started, and I got my license at 19 due to legal matters.

To this day, I sense that racing games were the ones that enabled me to
control aircrafts in an easier and faster manner than otherwise.

P.s. I've never played 3D flight simulator video games.

### Speed Dreams

During the years I've adopted open source, I've also looked into games
for the children, one of which was Speed Dreams (a project fork of
Torcs).

I was impressed, and my judgement is that Speed Dreams is the best
racing video game I've ever played, and probably one of the top 10
racing game simulators in history.

The mouse cursor ("gesture", so to speak) as a controller is one of the
factors that make Speed Dreams a good simulator.

I sense that mouse cursor as a controller is essential to STK and to
the skills and the patience development of children and youths.

**Comment by jwaxy** [comment]
Still no progress?

**Comment by qwertychouskie** [comment]
> Still no progress?

Nope, [as I said in May](https://github.com/supertuxkart/stk-code/issues/1832#issuecomment-1566750147):

> This is a pretty niche request, so I wouldn't hold your breath for the core dev team to work on it. If someone makes a PR with a good implementation of the feature, I would expect it to be merged, but otherwise, this feature is unlikely to happen.
